packages/utilities/gel-foundations/README.md

```diff
@@ -15,21 +19,27 @@ npm install @bbc/gel-foundations --save
 ```jsx
 import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
 
-import { GEL_GUTTER_BELOW_600PX } from '@bbc/gel-foundations/spacings';
+import { GEL_GUTTER_ABOVE_600PX } from '@bbc/gel-constants/spacings';
```

Comment:
I don't think this is right, `gel-constants` is deprecated

Reply:
Addressed below.